indico/modules/events/tracks/blueprint.py

```diff
@@ -25,4 +26,7 @@
 _bp.add_url_rule('/manage/tracks/', 'manage', RHManageTracks)
 _bp.add_url_rule('/manage/tracks/create', 'create', RHCreateTrack, methods=('GET', 'POST'))
 
+_bp.add_url_rule('/program/', 'program', RHDisplayTracks)
```

Comment:
I followed the same pattern as in `/contributions/` and `/contributions/contributions.pdf`, but I agree I like this better.